### ISSUE TITLE
Internal Exception Error

### ISSUE THREAD

**Issue by ladypluu** [Jomcraft-Network/DefaultSettings#10]
Hello!

When I attempt to connect to a server with DefaultSettings, I get this error:
[https://gyazo.com/a7790c9141d6a7e1f8eb92fb3f0dac6f](https://gyazo.com/a7790c9141d6a7e1f8eb92fb3f0dac6f)

It worked in singleplayer, but it stops me from connecting to a server.

**Comment by PT400C** [comment]
@ladypluu Hey, I can't reproduce that behaviour. Could you link/upload any further information that I could use to debug that for you? 

PT400C

**Comment by ladypluu** [comment]
Yeah sure! I have a latest.log here from when I get the error, but let me see if I can find anything else to help.
[latest.log](https://github.com/Jomcraft-Network/DefaultSettings/files/9248237/latest.log)

**Comment by PT400C** [comment]
From that log it seems, that your client's configs just don't match with what the server expects.

It would be helpful to have your modpack's folder and ideally also the files on the server's end to dive deeper into this.

**Comment by ladypluu** [comment]
@PT400C which files would you specifically need for the server? Just the mods/configs? I just want to make sure I'm giving you the stuff you need :)
Here's the modpack folder as well:
[Buppyverse 41.1.0.zip](https://github.com/Jomcraft-Network/DefaultSettings/files/9264202/Buppyverse.41.1.0.zip)
